examples/macosx-config.yml

```diff
@@ -20,7 +20,22 @@ config:
       userGroups: ['staff']
   logLevel:         info
   plugins:
-    disabled:       ['success', 'interactive']
+    disabled:       ['interactive']
+    env:
+      extra:
+        PATH: {$env: PATH}
+        TMPDIR: {$env: TMPDIR}
+        LANG: {$env: LANG}
+        LC_ALL: {$env: LC_ALL}
+        LOGNAME: {$env: LOGNAME}
+        TERM: {$env: TERM}
+        TERM_PROGRAM: {$env: TERM_PROGRAM}
+        TASK_ID: {$env: TASK_ID}
+        RUN_ID: {$env: RUN_ID}
+        TASKCLUSTER_WORKER_TYPE: {$env: TASKCLUSTER_WORKER_TYPE}
```

Comment:
WorkerType can probably be exposed through `TaskContext` instead... Or maybe `runtime.Environment` as passed in on plugin creation. Since this value doesn't change at runtime.